ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -24,10 +24,10 @@ public static string ValidYearLaterThanBirthYear(int birthYear)
         public const string PositiveNumbersOnly = "Please enter a positive value";
         public const string ContactTracingAdultsScreened = "Must be smaller or equal to the number of adults identified";
         public const string ContactTracingChildrenScreened = "Must be smaller or equal to the number of children identified";
-        public const string ContactTracingAdultsLatentTB = "Must be smaller or equal to than number of adults screened minus those with latent TB";
-        public const string ContactTracingChildrenLatentTB = "Must be smaller or equal to than number of children screened minus those with latent TB";
-        public const string ContactTracingAdultsActiveTB = "Must be smaller or equal to than number of adults screened minus those with active TB";
-        public const string ContactTracingChildrenActiveTB = "Must be smaller or equal to than number of children screened minus those with active TB";
+        public const string ContactTracingAdultsLatentTB = "Must be smaller or equal to than number of adults screened minus those with active TB";

```

Comment:
Latent and active are depend on each other therefore the messages should be the other way around